docs/user/gcp/install_upi.md

```diff
@@ -533,21 +530,28 @@ Create a resource definition file: `06_worker.yaml`
 $ cat <<EOF >06_worker.yaml
 imports:
 - path: 06_worker.py
-
 resources:
-- name: 'w-a-0'
+- name: 'w-0'
   type: 06_worker.py
   properties:
     infra_id: '${INFRA_ID}'
-    region: '${REGION}'
     zone: '${ZONE_0}'
-
     compute_subnet: '${COMPUTE_SUBNET}'
     image: '${CLUSTER_IMAGE}'
     machine_type: 'n1-standard-4'
     root_volume_size: '128'
     service_account_email: '${WORKER_SERVICE_ACCOUNT_EMAIL}'
-
+    ignition: '${WORKER_IGNITION}'
+- name: 'w-1'
+  type: 06_worker.py
+  properties:
+    infra_id: '${INFRA_ID}'
+    zone: '${ZONE_1}'
+    compute_subnet: '${COMPUTE_SUBNET}'
+    image: '${CLUSTER_IMAGE}'
+    machine_type: 'n1-standard-4'
+    root_volume_size: '128'
+    service_account_email: '${WORKER_SERVICE_ACCOUNT_EMAIL}'
```

Comment:
Why the extra worker?

Reply:
It's an example of how to do it, mostly.